transit_simulation/congestion_model.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+
+def voc(tod):
+    '''A helper function for defining the volume-capacity
+    ratio of theroad network'''
+    assert np.all((0 <= tod)&(tod <= 24)), "Time for defining congestion status should be given in hours since midnight"
+    
+    tod = np.asarray(tod)
+    voc = np.empty(tod.shape)
+    
+    voc[tod <= 24] = (tod[tod <= 24]-24) * -0.02

```

Comment:
24 is because there are 24 hours in a day. Other constants define the shape of the piecewise function modelling volume-to-capacity ratio of the Helsinki road network. Unfortunately the shape of the function cannot be easily modified changing these values, but it would need more changes to produce result that makes sense. Thus, it might not help to have all those in separate file, but I do not have experience of this so you can make changes if that is better!